assets/sass/libs/_vars.scss

```diff
@@ -40,10 +40,10 @@
 		fg-light:			rgba(255,255,255,0.35),
 		border:				rgba(255,255,255,0.15),
 		border-bg:			rgba(255,255,255,0.05),
-		accent1:			#5e42a6,
-		accent1-alt:		darken(#5e42a6, 10),
-		accent2:			#5052b5,
-		accent2-alt:		darken(#5052b5, 10),
+		accent1:			E7EBEE,
+		accent1-alt:		darken(E7EBEE, 10),
+		accent2:			#E7EBEE,
+		accent2-alt:		darken(#E7EBEE, 10),
 		accent3:			#b74e91,
 		accent3-alt:		darken(#b74e91, 10)
 	);
```

Comment:
This is still fine. Next time, do take it into account to create a newline for sass as it seems to be convention similar to Python.